test/e2e/diagram.overlays.rendering.test.ts

```diff
@@ -115,38 +159,75 @@ describe('BPMN elements with overlays', () => {
       customSnapshotIdentifier: 'remove.all.overlays.of.shape',
     });
   });
+});
 
-  // TODO tests will be added with https://github.com/process-analytics/bpmn-visualization-js/issues/1166
-  it.skip.each([<OverlayEdgePosition>'start', <OverlayEdgePosition>'middle', <OverlayEdgePosition>'end'])(
-    `add overlay on SequenceFlow on %s`,
-    async (position: OverlayEdgePosition) => {
+describe('BPMN Edges with overlays', () => {
+  const bpmnDiagramNameAssociations = 'overlays.edges.associations.complex.paths';
+  const bpmnDiagramNameMessageFlows = 'overlays.edges.message.flows.complex.paths';
+  const bpmnDiagramNameSequenceFlows = 'overlays.edges.sequence.flows.complex.paths';
+  function getEdgeKindAndIds(bpmnDiagramName: string): [string, string[]] {
```

Comment:
I would rather have made a map with the different values ​​instead.

Reply:
I am on it